docs/internal-port.md

```diff
@@ -35,6 +35,5 @@ metadata:
 ```
 
 ## Some limitation
-The internal port name should be unique on a host and kubelet always check the `eth0` interface in the Pod.
-
-To bypass this issue, Kube-OVN creates a dummy type device in Pod netns with the same ip address of internal port and set the eth0 down. It works well for most scenarios, however if applications rely on network interface name, it will bring confusions.
+- The internal port name should be unique on a host and kubelet always check the `eth0` interface in the Pod. To bypass this issue, Kube-OVN creates a dummy type device in Pod netns with the same ip address of internal port and set the eth0 down. It works well for most scenarios, however if applications rely on network interface name, it will bring confusions.
```

Comment:
The internal port name `must` be unique on a host and kubelet always `checks` the `eth0` interface in the Pod. To bypass this issue, Kube-OVN creates a dummy interface named `eth0` in the Pod's netns, assigns the same IP address(es), and sets it down.